### ISSUE TITLE
Confusing naming in Chapter/Figure 20.3

### ISSUE THREAD

**Issue by Mr-Pine** [remzi-arpacidusseau/ostep-typos#96]
Chapter 20.3 calls the addresses stored in a page directory entry `Page Frame Number (PFN)`. In my opinion this is really confusing as this value does not identify a page frame but a page *table*, so calling it a `Page Table Number` or something similar would be a lot clearer. This is especially true in the figure where there are `PFN` labels both at the top and on the side of the page directory/table(s) that mean completely different things.

**Comment by remzi-arpacidusseau** [comment]
yeah thanks. Unfortunately these are the terms people use and hence we use them. Will think about ways to clarify ...
